js/rendererjs/disabledplugin.js

```diff
@@ -28,10 +28,15 @@ const handleReport = () => {
 	shell.openExternal('https://github.com/NebulousLabs/Sia/issues')
 }
 
-
-const DisabledPlugin = ({errorMsg, startSiad}) => (
+const DisabledPlugin = ({ errorMsg, startSiad }) => (
 	<div style={containerStyle}>
-		<h2>Siad has exited unexpectedly. Please submit a bug report including the error log <a style={reportStyle} onClick={handleReport}>here.</a></h2>
+		<h2>
+			Siad has exited unexpectedly. Please submit a bug report including the
+			error log{' '}
+			<a style={reportStyle} onClick={handleReport}>
+				here.
+			</a>
+		</h2>
```

Comment:
hmm. Does this affect the actual display of this element? I feel like a JS formatter shouldn't be touching HTML.

Reply:
note that this is jsx, not html. although it looks similar, there are some differences to how it works. It wouldn't affect the actual display of the element because jsx elements transpiles into javascript objects.